docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -19,6 +19,116 @@ If applicable, exact error messages linked to this issue so users searching for
 Link to fix
 ////
 
+[[broken-apm-anomaly-rule]]
+*Upgrading to v8.13 breaks APM anomaly rules* +
+_Elastic Stack versions: 8.13.0, 8.13.1, 8.13.2_ +
+_Fixed in Elastic Stack version 8.14.0_
+
+// The conditions in which this issue occurs
+This issue occurs when upgrading the Elastic Stack to version 8.13.0 or 8.13.1.
+This issue may go unnoticed unless you actively monitor your {kib} logs.
+The following log indicates the presence of this issue:
+[source,shell]
+----
+"params invalid: [anomalyDetectorTypes]: expected value of type [array] but got [undefined]"
+----
+
+This issue occurs because a non-optional parameter, `anomalyDetectorTypes` was added in 8.13.0 without
+the presence of an automation migration script. This breaks pre-existing rules as they do not have this parameter
+and will fail validation. This issue is fixed in v8.14.0.
+
+It is possible to fix this error without upgrading. Follow these steps:
+
+. Find broken rules
++
+====
+To identify rules in this exact state, you can use the {kibana-ref}/find-rules-api.html[find endpoint] and search for the APM anomaly rule type as well as this exact error message indicating that the rule is in the broken state. We will also use the `fields` parameter to specify only the fields required when making the update request later.
```

Comment:
```suggestion
To identify rules in this exact state, you can use the {kibana-ref}/find-rules-api.html[find rules endpoint] and search for the APM anomaly rule type as well as this exact error message indicating that the rule is in the broken state. We will also use the `fields` parameter to specify only the fields required when making the update request later.
```
Not sure if the rules part is necessary or not, but thought it might add clarity.